Library/Homebrew/os/mac.rb

```diff
@@ -41,6 +41,10 @@ def cat
       version.to_sym
     end
 
+    def language
+      @language ||= `defaults read .GlobalPreferences AppleLanguages`.delete(" \n\"()").sub(%r{,.*}, '')
```

Comment:
Can we use `Utils.popen_read` instead here? Backticks are inconsistent with the rest of the file here... and kind of gross to look at 😄.